Solved Ethernet i225-V with a Kernel Patch

RobyRew opened the pull request:
As you can [see here](https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#patch-2) the Ethernet card for this MoBo can be now used without a kext, just a simple kernel patch

`				<key>Base</key>`
`				<string>__Z18e1000_set_mac_typeP8e1000_hw</string>`
` 				<key>Comment</key>`
` 				<string>Fixing I225-V Ethernet</string>`
` 				<key>Enabled</key>`
` 				<true/>`
` 				<key>Find</key>`
` 				<data>8hUAAA==</data>`
` 				<key>Identifier</key>`
` 				<string>com.apple.driver.AppleIntelI210Ethernet</string>`
` 				<key>MinKernel</key>`
` 				<string>19.0.0</string>`
` 				<key>Replace</key>`
` 				<data>8xUAAA==</data>`

I deleted the kext, and deleted from config.plist, now its clean under Kernel>Patch

g3d commented:
@SchmockLord can you check if this one should be included as well?

SchmockLord commented:
@g3d This is just another way of getting the i225-V supported. But: This only works for people with Big Sur 11.3 or newer. And in the past there were always people that use an older macOS. And as soon as I applied fixes like this those people are raising issues like "I225-V not working with your current release". And I try to avoid that. The IntelMausiEthernet.kext should be working for older and newer macOS versions.

g3d commented:
@SchmockLord should we close it then?

wutaijieing commented:
1

> As you can [see here](https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#patch-2) the Ethernet card for this MoBo can be now used without a kext, just a simple kernel patch
> 
> ` <key>Base</key>` ` <string>__Z18e1000_set_mac_typeP8e1000_hw</string>` ` <key>Comment</key>` ` <string>Fixing I225-V Ethernet</string>` ` <key>Enabled</key>` ` <true/>` ` <key>Find</key>` ` <data>8hUAAA==</data>` ` <key>Identifier</key>` ` <string>com.apple.driver.AppleIntelI210Ethernet</string>` ` <key>MinKernel</key>` ` <string>19.0.0</string>` ` <key>Replace</key>` ` <data>8xUAAA==</data>`
> 
> I deleted the kext, and deleted from config.plist, now its clean under Kernel>Patch

I225-v how to add wake on LAN？

RobyRew commented:
> 1
> 
> > As you can [see here](https://dortania.github.io/OpenCore-Install-Guide/config.plist/comet-lake.html#patch-2) the Ethernet card for this MoBo can be now used without a kext, just a simple kernel patch
> > ` <key>Base</key>` ` <string>__Z18e1000_set_mac_typeP8e1000_hw</string>` ` <key>Comment</key>` ` <string>Fixing I225-V Ethernet</string>` ` <key>Enabled</key>` ` <true/>` ` <key>Find</key>` ` <data>8hUAAA==</data>` ` <key>Identifier</key>` ` <string>com.apple.driver.AppleIntelI210Ethernet</string>` ` <key>MinKernel</key>` ` <string>19.0.0</string>` ` <key>Replace</key>` ` <data>8xUAAA==</data>`
> > I deleted the kext, and deleted from config.plist, now its clean under Kernel>Patch
> 
> I225-v how to add wake on LAN？

Should be enabled on BIOS
